### PR TITLE
My Week 2 Pull Request

### PR DESCRIPTION
**Pull request by markatron9000**
Howdy!
This is my week 2 pull request - I will be adding to it slowly, but here it is! Thank you!

### CODE REVIEW

**Comment by markatron9000** [comment]
Ayo

File: portfolio/src/main/java/com/google/sps/servlets/buildtable.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return newt;
+    }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =  Query.newEntityQueryBuilder().setKind("Bask")
+            .setOrderBy(OrderBy.desc("TimestampForMusic"))
+            .build();
+            //Query<Entity> query = new Query<Entity>("Music Selection");
```
Comment (claireyaywu) [inline portfolio/src/main/java/com/google/sps/servlets/buildtable.java:33]: remove dead code

File: portfolio/src/main/java/com/google/sps/servlets/buildtable.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        ArrayList<String> songs =  new ArrayList<>();
+        while (results.hasNext()) {
+            Entity entity = results.next();
+            String song = entity.getString("MusicSelection");
+                
+            songs.add(song);
+        }
+        Gson bson = new Gson();
+        response.setContentType("application/json;");
+        //String toReturn = convertToJsonUsingGson(songs);
```
Comment (claireyaywu) [inline portfolio/src/main/java/com/google/sps/servlets/buildtable.java:49]: ditto

File: portfolio/src/main/webapp/serverletscripts.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  const dateContainer = document.getElementById('forServlet');
+  dateContainer.innerHTML = textFromResponse;
+}
+
+async function getListOfQuotes(){
+  const responseFromServer = await fetch('/quotesServlet');
+
+  const quotesList = await responseFromServer.json();
+  //below ones won't represent in json, due to extremely special characters
```
Comment (claireyaywu) [inline portfolio/src/main/webapp/serverletscripts.js:13]: thanks for the comment

**Review by claireyaywu — APPROVED** [review]
this PR is way too big. (next time i would ask you to break it down to a few small ones) a good practice is to always keep your PRs small.

File: portfolio/src/main/java/com/google/sps/servlets/buildtable.java (hunk 1 above)
Comment (markatron9000) [reply portfolio/src/main/java/com/google/sps/servlets/buildtable.java:33]: Okay, will do. And i apologize about the size of the pull request - back in week 2 when I made this, i didn't know that pull requests were supposed to be small, lol

**Pull request merged**